<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialogClassDialog</class>
 <widget class="QDialog" name="NewDialogClassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Dialog Class</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <layout class="QGridLayout" name="_3">
     <item row="2" column="1">
      <widget class="QLineEdit" name="pathnameEdit">
       <property name="toolTip">
        <string>Enter the path of the file for the forms code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Classname:</string>
       </property>
       <property name="buddy">
        <cstring>classnameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Filename:</string>
       </property>
       <property name="buddy">
        <cstring>filenameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Path:</string>
       </property>
       <property name="buddy">
        <cstring>pathnameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="classnameEdit">
       <property name="toolTip">
        <string>Enter the name of the new class</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="filenameEdit">
       <property name="toolTip">
        <string>Enter the name of the file for the forms code</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="pathButton">
       <property name="toolTip">
        <string>Select the source file path via a directory selection dialog</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>classnameEdit</tabstop>
  <tabstop>filenameEdit</tabstop>
  <tabstop>pathnameEdit</tabstop>
  <tabstop>pathButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDialogClassDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDialogClassDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
